### PR TITLE
Make ConnectedStatusIndicator clickable

### PR DESCRIPTION
**Pull request by whymarrh**
This PR adds an `onClick` prop to the `ConnectedStatusIndicator`, making it a button

### CODE REVIEW

File: ui/app/components/app/connected-status-indicator/index.scss [hunk 1]
```diff
@@ -3,6 +3,11 @@
   align-items: center;
   place-self: center start;
 
+  background: none;
+  font-size: inherit;
+  padding: 0;
+  cursor: pointer;
```
Comment (Gudahtt) [inline ui/app/components/app/connected-status-indicator/index.scss:9]: Nit: is this necessary? Buttons should already have a pointer cursor style.

**Review by Gudahtt — DISMISSED** [review]
Generally this change looks good, but this isn't a very good button at the moment. It doesn't change on hover/focus as buttons typically do, and the hitbox is pretty small.

**Comment by whymarrh** [comment]
Removed the extraneous `cursor` rule and left a bit of top+bottom padding for a larger hitbox

**Comment by whymarrh** [comment]
> It doesn't change on hover/focus as buttons typically do

Yep, but this is better than nothing. This is something we need to address with all of our buttons.

**Comment by Gudahtt** [comment]
Our other buttons already do that though (via the `:active` pseudo-class). Well, anything that uses our base `Button` class does anyway. We do have a few scattered uses of `button` without this (like the Popover close button).

**Review by Gudahtt — APPROVED** [review]

**Pull request merged**